actor-tests/src/test/scala/org/apache/pekko/actor/ActorSystemSpec.scala

```diff
@@ -392,6 +392,22 @@ class ActorSystemSpec extends PekkoSpec(ActorSystemSpec.config) with ImplicitSen
         }
       } finally shutdown(sys)
     }
+    "not include username in toString" in {
+      // Actor System toString is output to logs and we don't want env variable values appearing in logs
+      val system =
+        ActorSystem(
+          "config-test-system",
+          ConfigFactory
+            .parseString("""pekko.test.java.property.home = "${user.home}"""")
+            .withFallback(PekkoSpec.testConf))
+      val debugText = system.settings.toString
+      val username = System.getProperty("user.name")
+      val userHome = System.getProperty("user.home")
+      (debugText should not).include(username)
+      (debugText should not).include(userHome)
+      debugText should include("<username>")
+      shutdown(system)
```

Comment:
Minor nit, you can put this into a `try`/`finally` block so the actor system is shutdown even if an exception is thrown